### DISCUSSION TITLE
Invalid Message Type

### DISCUSSION THREAD

**Discussion by mmarkmm** [enisdenjo/graphql-ws#421] [Q&A] [answered]
Hello @all,

I am using this link.
https://unpkg.com/graphql-ws/umd/graphql-ws.min.js

{"type":"connection_error","payload":"Could not verify JWT: JWTExpired"}

Error: Invalid message 'type' property "connection_error"
    at i (graphql-ws.min.js:1:3264)
    at p (graphql-ws.min.js:1:3460)
    at i.onmessage (graphql-ws.min.js:1:7667)

There is no such type in the file, do you have any suggestions for this?

**Comment by enisdenjo** [comment, marked as answer]
Hi there, your server is using the [deprecated `subscriptions-transport-ws`](https://github.com/apollographql/subscriptions-transport-ws).

[This library is not backwards compatible.](https://github.com/enisdenjo/graphql-ws#disclaimer)

Please consider updating your server to use `graphql-ws` too or implement the [GraphQL over WebSocket Protocol](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md).

**Comment by mmarkmm** [comment]
Hello @enisdenjo , thank you for your update.
